src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/HiddenFieldCheck.java

```diff
@@ -144,19 +183,21 @@
  * &lt;/module&gt;
  * </pre>
  * <pre>
- * class SomeClass
- * {
- *   private List&lt;String&gt; test;
+ * public class SomeClass {
  *
- *   private void addTest(List&lt;String&gt; test) // no violation
- *   {
- *     this.test.addAll(test);
- *   }
+ *   private String field;
+ *   private String testField;
  *
- *   private void foo()
- *   {
- *     final List&lt;String&gt; test = new ArrayList&lt;&gt;(); // no violation
- *     ...
+ *   public SomeClass(String testField) { // OK, 'testField' param doesn't hide any field
```

Comment:
in default config example it does hide, here it is not. Are you sure ?
can you prove by CLI that this example works like it is described in comments?

you should change `value=&quot;^test$&quot;/&gt;` to `value=&quot;^testField$&quot;/&gt;`
to make test resonable.
and comment should be "// OK, because it match ignoreFormat"

Reply:
> comment should be "// OK, because it match ignoreFormat"

should i make all the `// OK` comments similar to this?

Reply:
done 
and i forgot that, sorry.